websites/S/Sonic Retro/presence.ts

```diff
@@ -0,0 +1,279 @@
+const presence = new Presence({
+		clientId: "970743721404530798"
+	}),
+	browsingTimestamp = Math.floor(Date.now());
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname, search, hostname } = window.location;
+	switch (hostname) {
+		case "sonicretro.org": {
+			presenceData.smallImageKey = "home_512";
+			presenceData.smallImageText = "sonicretro.org";
```

Comment:
This isn't needed

Reply:
> This isn't needed

Which part, the image or the URL?

Reply:
the URL on the small image text

Reply:
> the URL on the small image text

Should the others go too?

Reply:
> I don’t understand what you mean

The other `smallImageText`'s for the other cases

Reply:
👀

Reply:
this wasn't addressed. Yes you should remove all smallImageTexts that contain a URL

Reply:
All right that was all I was asking, but u never responded so I resolved to get ur attention on the matter. Will fix when I get to a computer.